releases/pulpcore/3.6/settings

```diff
@@ -0,0 +1,3 @@
+FOREMAN_VERSION='2.1'
```

Comment:
if I don't set that, the scripts bail out, and adding "if not pulpcore" seemed also weird?

Reply:
Do we need to inherit anything? Perhaps we can support `FOREMAN_VERSION=''` or `FOREMAN_VERSION='none'`.

Reply:
I've implemented "none" now, that was a good idea.